tests/LocalDriverTest.php

```diff
@@ -60,6 +60,18 @@ public function testCreateClassConfigEmpty()
         new LocalDriver();
     }
 
+    public function testCreateDirectoryIfNotExists()
+    {
+        $productionPath = __DIR__ . '/../storage/non_existent_directory/documentation.json';
+        config(['auto-doc.drivers.local.production_path' => $productionPath]);
+
+        mkdir(dirname($productionPath), 0777, true);
```

Comment:
directory should be created by the testable class, not via the native php function